administration/package.json

```diff
@@ -101,7 +112,7 @@
     "ts:check": "tsc --build",
     "format": "NODE_ENV=test eslint . --fix && prettier . --write",
     "generate-graphql": "graphql-codegen --config graphql-codegen.yml",
-    "generate-protobuf": "protoc -I ../specs --es_out=src/generated --es_opt=target=ts --plugin=protoc-gen-es=./node_modules/.bin/protoc-gen-es ../specs/card.proto",
+    "generate-protobuf": "protoc -I ../specs --es_out=src/generated --es_opt=target=ts --plugin=protoc-gen-es=../node_modules/.bin/protoc-gen-es ../specs/card.proto",
```

Comment:
if it only for me the new command doesn't work?

Reply:
Hmm. I don't remember changing this, maybe this was just a mistake or done automatically for some reason? Please feel free to revert this, sorry.

Reply:
well, it's failing in the ci otherwise

Reply:
Did you try removing node_modules and reinstalling?

Reply:
no, I decided to complain first 😄
ok, I see now, the folder has been moved from administration to the project root, thanks